.github/workflows/build_and_test.yml

```diff
@@ -1231,7 +1231,7 @@ jobs:
           payload: |
             channel: ${{ vars.SLACK_STORAGE_CHANNEL_ID }}
             text: |
-              🔴 @oncall-storage: deploy job on release branch had unexpected status "${{ needs.deploy.result }}" <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|GitHub Run>.
+              🔴 <!subteam^S06CJ87UMNY>: deploy job on release branch had unexpected status "${{ needs.deploy.result }}" <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|GitHub Run>.
```

Comment:
Let's add a comment with the team name?

Reply:
Replaced `<!subteam^S06CJ87UMNY>` with `<!subteam^S06CJ87UMNY|@oncall-storage>`. The resulting slack message is the same.